[2024-04-19T19:21:32.240+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-19T19:21:32.270+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: model_pipeline.load_data manual__2024-04-19T19:21:28.177901+00:00 [queued]>
[2024-04-19T19:21:32.282+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: model_pipeline.load_data manual__2024-04-19T19:21:28.177901+00:00 [queued]>
[2024-04-19T19:21:32.283+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-19T19:21:32.302+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data> on 2024-04-19 19:21:28.177901+00:00
[2024-04-19T19:21:32.313+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1226) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-19T19:21:32.315+0000] {standard_task_runner.py:63} INFO - Started process 1235 to run task
[2024-04-19T19:21:32.315+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'model_pipeline', 'load_data', 'manual__2024-04-19T19:21:28.177901+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline_dag.py', '--cfg-path', '/tmp/tmp75ninxom']
[2024-04-19T19:21:32.317+0000] {standard_task_runner.py:91} INFO - Job 83: Subtask load_data
[2024-04-19T19:21:32.377+0000] {task_command.py:426} INFO - Running <TaskInstance: model_pipeline.load_data manual__2024-04-19T19:21:28.177901+00:00 [running]> on host ec31c9b90297
[2024-04-19T19:21:32.481+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='model_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-19T19:21:28.177901+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-19T19:21:28.177901+00:00'
[2024-04-19T19:21:32.484+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-19T19:21:33.597+0000] {python.py:237} INFO - Done. Returned value was:        rev_Mean  mou_Mean  totmrc_Mean  ...  ethnic_U  ethnic_X  ethnic_Z
0       31.6625     25.50      29.9900  ...         0         0         0
1       62.1275     97.50      65.9850  ...         0         0         0
2       32.0550    137.75      30.0000  ...         0         0         0
3       26.0450   1347.00      10.0000  ...         0         0         0
4       71.8575    537.50      75.0000  ...         0         0         0
...         ...       ...          ...  ...       ...       ...       ...
99995   60.5675   1077.00      60.0725  ...         0         0         0
99996   17.1450      4.50      16.9900  ...         0         0         0
99997   34.9800     66.25      34.9800  ...         0         0         0
99998   60.1775    334.25      48.5700  ...         0         0         0
99999   30.3050    219.75      39.9900  ...         0         0         0

[100000 rows x 134 columns]
[2024-04-19T19:21:33.781+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-19T19:21:36.159+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=model_pipeline, task_id=load_data, execution_date=20240419T192128, start_date=20240419T192132, end_date=20240419T192136
[2024-04-19T19:21:36.199+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-19T19:21:36.233+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-19T19:21:36.236+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
